### ISSUE TITLE
Unable to migrate from xteve

### ISSUE THREAD

**Issue by foux** [Threadfin/Threadfin#389]
**Describe the bug**
I'm trying to migrate from xteve. Following https://github.com/Threadfin/Threadfin/issues/148, I created a backup off xteve, and unzipped it to Threadfin conf folder. But when starting threadfin, I get the error `The database version of your settings is not compatible with this version`

**Server (please complete the following information):**
- OS Unraid - Docker
- Arch Intel 64 bits
- Threadfin Version and Build 1.2 Build: 8

**To Reproduce**
1. Create a backup of xteve 2.2.0 config
2. Export this backup to Threadfin root
3. Start threadfin

**Expected behavior**
Threadfin should start with xteve settings

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome

**Comment by Fyb3roptik** [comment]
This is not a thing anymore. You will need to recreate it

**Comment by foux** [comment]
> This is not a thing anymore. You will need to recreate it

I don't understand your answer? What do you mean I'll need to recreate? You mean it isn't possible to migrate from xteve to Threadfin?
